### ISSUE TITLE
Upload file action keeps on loading

### ISSUE THREAD

**Issue by Harshil-Modi** [FlutterFlow/flutterflow-issues#801]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When file upload action is called and file selector window is closed by pressing `Esc`, then the button with the action is still in loading state.

### Expected Behavior

The button with file upload shouldn't be in loading state.

### Steps to Reproduce

1. Add `Upload media` action on a widget.
2. Run in debug mode.
3. Click on the widget with `Upload media` action.
4. Once, the popup is opened to select file, press `Esc`.
5. Observe the widget with action

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESk876y4Vgodtf7arAdsBAhQAkH1x/bZJEl9cbaAAdB772ELMifer+VFZYT+m7YXw7PlGink8y0/zOjoaeDPYnGxacaINo1ZNcSAvjfHGmeKaXPpGoYnBQTP9UGH2O5Kevvx5DHLNsSmQew3z/cq3qNleeY8aSfxBlZ7vfcPo=

### Context

I was in demo with my client, and wanted to show upload media. It didn't affect me on any means, but user experience wise it will a bit odd. 

### Visual documentation

![Test-Mode---exps](https://user-images.githubusercontent.com/31483641/233777064-799487a6-897d-4a62-9526-c8a0e8f0f798.gif)

### Environment

```markdown
- FlutterFlow version: 3.7.3 
- Platform: Web
- Browser name and version: Chrome 112.0.5615.138 (Official Build) (64-bit) (cohort: Stable) 
- Operating system and version affected: Windows 10
```

**Comment by wenkaifan0720** [comment]
Hi @Harshil-Modi. I tried with Chrome on MacOS and it seems to be working:

https://user-images.githubusercontent.com/112435854/234108775-99f5ef8c-e39e-4106-83ba-92651cbf5df2.mp4

I also tried with running it with an Android emulator and it is also working as expected. It might be due to the implementation of the windows file picker (Also this test mode app is wrapped in an iFrame). Since this is not affecting the generated app running on real devices (things will not 100% work in test mode, see known issues), and we don't have full support on Windows, I'll flag this as low priority.
